### ISSUE TITLE
Could you please provide the test set CSV file of the Ningbo dataset?

### ISSUE THREAD

**Issue by kaivin321** [Edoar-do/HuBERT-ECG#14]
Hi Edoardo,

I would like to try fine-tuning and inference on the Ningbo dataset to evaluate the model’s performance. However, I noticed that the reproducibility folder only contains the train and validation CSVs for cross-validation, but no test CSV. I kindly request you to add a ningbo test set CSV. Thank you very much!

**Comment by Edoar-do** [comment]
Hey @kaivin321 , I must've missed that! Thank you for noticing!
Now you can find it in the corresponding folder

**Comment by kaivin321** [comment]
Thanks for adding it! Just saw it in the folder

**Comment by Edoar-do** [comment]
Issue solved
